src/Microsoft.DotNet.Build.Tasks/Targets/packages.targets

```diff
@@ -61,7 +61,7 @@
 
     <GetPackageVersion
       Condition="Exists('$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll')"
-      DailyBuildNumber="$(DailyBuildNumber)"
+      BuildNumber="$(BuildNumber)"
```

Comment:
I thought about this but I couldn't think of a scenario where this should legitimately not be set.  If there is I can certainly add a default value.